plugins/contextinfo/PLUGININFO

```diff
@@ -3,4 +3,4 @@ Authors=['Guillaume Lecomte <guillaume86@gmail.com>']
 Name=_('Contextual Info')
 Description=_('Show various informations about the track currently playing.\nDepends: libwebkit >= 1.0.1, python-webkit >= 1.1.2, python-imaging (a.k.a. PIL)')
 Category=_('Information')
-RequiredModules=['PIL', 'webkit']
+RequiredModules=['PIL']
```

Comment:
Did you mean extending the `RequiredModules` functionality to add gi:WebKit2? How about adding another field to PLUGININFO, e.g. `RequiredGiModules`? It would be useful to add another one too for dbus services (for plugins awn, currentsong, …), e.g. call it `RequiredDbusServices`.